controller-server/src/main/java/com/yahoo/vespa/hosted/controller/maintenance/RoutingPolicies.java

```diff
@@ -161,7 +168,7 @@ private void removeObsoleteEndpointsFromDns(LoadBalancers loadBalancers, @Suppre
         Set<RoutingId> routingIds = new LinkedHashSet<>();
         for (var loadBalancer : loadBalancers) {
             for (var rotation : loadBalancer.rotations()) {
-                routingIds.add(new RoutingId(loadBalancer.application(), rotation));
+                routingIds.add(new RoutingId(loadBalancer.application(), EndpointId.of(rotation.value())));
```

Comment:
This seems odd. `RotationName` and `EndpointId` are different things, right?

Reply:
It's temporary to reduce dependencies on `RotationName`. They represent the same thing.